pkgs/desktops/gnome/games/aisleriot/default.nix

```diff
@@ -61,9 +61,8 @@ stdenv.mkDerivation rec {
   ];
 
   passthru = {
-    updateScript = gnome.updateScript {
-      packageName = pname;
-      attrPath = "gnome.${pname}";
+    updateScript = nix-update-script {
+      attrPath = pname;
```

Comment:
The attribute is still in the gnome namespace.

Reply:
You mean that `attrPath` should still have the value `gnome.${pname}` ?

Reply:
Yeah. Or we need to move aisleriot to all-packages.nix.

Reply:
Ok I'll apply that change, I think we should still keep it under gnome for now, even though from what I see in the package definition, it can be built in a kde-frendly way.

Reply:
I have applied the change. Thanks for the heads up